evaluation/benchmarks/aider_bench/run_infer.py

```diff
@@ -60,6 +61,7 @@ def get_config(
         workspace_base=None,
         workspace_mount_path=None,
     )
+    config = override_openhands_config_for_eval(config)
```

Comment:
@OpenHands make sure you replace all occurence of override_openhands_config_for_eval with get_openhands_config_for_eval